src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -64,75 +64,69 @@ class CloseOrderModal extends Component {
-    const isVisible = orderId && workflowStatus === WORKFLOW_STATUS.open;
+    const { orderNumber, closingReasons, closeOrderSubmit, onCancel } = this.props;
     const reasons = closingReasons.map((reason) => ({
       label: reason.value,
       value: reason.value,
     }));
 
-    if (!isVisible) {
-      return null;
-    }
-
     return (
-      <Fragment>
-        <Button
-          buttonStyle="primary"
-          marginBottom0
-          onClick={this.openModal}
-          style={{ marginRight: '10px' }}
-        >
-          <FormattedMessage id="ui-orders.paneBlock.closeBtn" />
-        </Button>
-        <Modal
-          label={<FormattedMessage id="ui-orders.closeOrderModal.title" values={{ orderId }} />}
-          open={this.state.openModal}
-        >
-          <Row>
-            <Col xs={12}>
-              <Select
-                dataOptions={reasons}
-                label={<FormattedMessage id="ui-orders.closeOrderModal.reason" />}
-                onChange={e => this.setState({ reason: e.target.value })}
-                required
-              >
-                {Object.keys(DEFAULT_REASONS).map((key) => (
-                  <FormattedMessage
-                    id={`ui-orders.closeOrderModal.closingReasons.${key}`}
-                    key={key}
-                  >
-                    {(message) => <option value={DEFAULT_REASONS[key]}>{message}</option>}
-                  </FormattedMessage>
-                ))}
-              </Select>
-              <TextArea
-                label={<FormattedMessage id="ui-orders.closeOrderModal.notes" />}
-                onChange={e => this.setState({ note: e.target.value })}
-              />
-            </Col>
-          </Row>
-          <Row>
-            <Col
-              className={css.buttonsLine}
-              xs={12}
+      <Modal
+        label={<FormattedMessage id="ui-orders.closeOrderModal.title" values={{ orderNumber }} />}
+        open={this.state.openModal}
+      >
+        <Row>
+          <Col xs={12}>
+            <Select
+              autoFocus
+              label={<FormattedMessage id="ui-orders.closeOrderModal.reason" />}
+              onChange={e => this.setState({ reason: e.target.value })}
+              placeholder=" "
+              defaultValue=""
+            >
+              {Object.keys(DEFAULT_REASONS).map((key) => (
+                <FormattedMessage
+                  id={`ui-orders.closeOrderModal.closingReasons.${key}`}
+                  key={key}
+                >
+                  {(message) => <option value={DEFAULT_REASONS[key]}>{message}</option>}
+                </FormattedMessage>
+              ))}
+              {reasons.map((index) => (
+                <option value={index.label}>{index.value}</option>
+              ))}
+            </Select>
+            <TextArea
+              label={<FormattedMessage id="ui-orders.closeOrderModal.notes" />}
+              onChange={e => this.setState({ note: e.target.value })}
```

Comment:
I think it's a best practice to extract handlers to class methods, so the final version should look like `onChange={this.noteChanged}`